src/presentational/AssignmentForm.jsx

```diff
@@ -53,11 +53,36 @@ class AssignmentForm extends Component {
         newAssignment[key] = this.state[key];
       }
     }
+    
+    let savedSubjectsIDs = sessionStorage.getItem('savedSubjectsIDs');
```

Comment:
there's a few instances of this - check out [this link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON#Methods) for how to use `JSON.stringify` and `JSON.parse`